### ISSUE TITLE
Splitter should be moveable using keyboard

### ISSUE THREAD

**Issue by taauntik** [bryntum/support#11914]
[Forum post](https://forum.bryntum.com/viewtopic.php?f=51&t=34065&p=173744#p173744)

Hi Team,
On the Gantt, there’s a splitter between panels, but it doesn’t seem possible to move it using the keyboard. Could you please suggest if there’s a supported way to enable keyboard interaction for resizing or repositioning the splitter? or consider this as a keyboard accessibility issue.

Example:
https://bryntum.com/products/gantt/examples/bigdataset/

**Comment by Lalaluka** [comment]
Could we (@telekom) get an Update on this issue? It also showed up in our Accessibility Assessment.
While we could add focus to other Bryntum Splitters in our Application, we were not able to do that for the Splitter inside the Scheduler yet.

**Comment by ExtAnimal** [comment]
The solution, I propose, is for the Splitter component to make its encapsulating element tabbable, ie make it `tabIndex="0"`

Then add `keyMap` entries dynamically depending on its orientation. If horizontal, then `ArrowUp`/`ArrowDown` should move the Splitter. If vertical, then `ArrowLeft`/`ArrowRight` should move the Splitter.

There needs to be a meaningful, localized `ariaLabel` describing the Component when focused.

The `KeyMap` mixin should keep the `aria-keyshortcuts`: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-keyshortcuts attribute updated.

The speed of movement on each keystroke needs to be configurable. 1px on each repeat of the keydown would feel too slow.
